distribution/docs/src/main/resources/_contents/_endpoints/csw-endpoint-contents.adoc

```diff
@@ -0,0 +1,2118 @@
+                <ogc:ComparisonOperator>GreaterThanEqualTo</ogc:ComparisonOperator>
+                <ogc:ComparisonOperator>LessThan</ogc:ComparisonOperator>
+                <ogc:ComparisonOperator>LessThanEqualTo</ogc:ComparisonOperator>
+                <ogc:ComparisonOperator>EqualTo</ogc:ComparisonOperator>
+                <ogc:ComparisonOperator>NotEqualTo</ogc:ComparisonOperator>
+            </ogc:ComparisonOperators>
+        </ogc:Scalar_Capabilities>
+        <ogc:Id_Capabilities>
+            <ogc:EID/>
+        </ogc:Id_Capabilities>
+    </ogc:Filter_Capabilities>
+</csw:Capabilities>
+----
+
+.`DescribeRecord` Operation
+The `describeRecord` operation retrieves the type definition used by metadata of one or more registered resource types.
+There are two request types one for `GET` and one for `POST`.
+Each request has the following common data parameters:
+
+Namespace:: In `POST` operations, namespaces are defined in the xml. In `GET` operations, namespaces are defined in a comma separated list of the form: `xmlns([prefix=]namespace-url)(,xmlns([prefix=]namespace-url))*`
+Service:: The service being used, in this case it is fixed at CSW.
+Version:: The version of the service being used (2.0.2).
+OutputFormat:: The requester wants the response to be in this intended output. Currently, only one format is supported (application/xml). If this parameter is supplied, it is validated against the known type. If this parameter is not supported, it passes through and returns the XML response upon success.
+SchemaLanguage: The schema language from the request. This is validated against the known list of schema languages supported (refer to http://www.w3.org/XML/Schema).
+
+.`DescribeRecord` HTTP GET
+The `HTTP GET` request differs from the `POST` request in that the typeName is a comma-separated list of namespace prefix qualified types as strings (e.g., csw:Record,xyz:MyType).
+These prefixes are then matched against the prefix qualified namespaces in the request.
+This is converted to a list of QName(s).
+In this way, it behaves exactly as the post request that uses a list of QName(s) in the first place.
+
+.`DescribeRecord` KVP Encoding
+----
+${secure_url}/services/csw?service=CSW&version=2.0.2&request=DescribeRecord&NAMESPACE=xmlns(http://www.opengis.net/cat/csw/2.0.2)&outputFormat=application/xml&schemaLanguage=http://www.w3.org/XML/Schema
+----
+
+.`DescribeRecord` HTTP POST
+The HTTP POST request `DescribeRecordType` has the `typeName` as a List of QName(s). The QNames are matched against the namespaces by prefix, if prefixes exist.
+￼￼￼￼
+.`DescribeRecord` XML Request
+[source,xml,linenums]
+----
+<?xml version="1.0" ?>
+  <DescribeRecord
+    version="2.0.2"
+    service="CSW"
+    outputFormat="application/xml"
+    schemaLanguage="http://www.w3.org/XML/Schema"
+    xmlns="http://www.opengis.net/cat/csw/2.0.2">
+  </DescribeRecord>
+----
+
+.`DescribeRecord` Sample Response (`application/xml`)
```

Comment:
Same comment as above, i.e., difficult to read in PDF format. Should we link to an external example, move to an appendix and refer to it from here, or remove altogether?